.github/workflows/ci_python.yaml

```diff
@@ -126,8 +126,10 @@ jobs:
           --cov=lamp_py
 
       # Upload Coverage as an Artifact for Subsequent Jobs
+      - name: Setup LCOV
+        uses: hrishikesh-kadam/setup-lcov@v1
```

Comment:
Can you try removing this and see if the `pytest` CI is successful? I can't see why would need this added based on the changes from v1 to v4 in the `mbta/github-actions-report-lcov` action.

Reply:
Ok, looks like that setup IS needed! Thanks for checking, you can disregard this one.